src/Constants.js

```diff
@@ -15,7 +22,155 @@ export const DEFAULT_QUERY = `{
+            -104,
+            41
+         ],
+         [
+            -105.251255,
+            41
+         ]
+      ]
+   ]}`;
+
+const DEFAULT_DECODE_BODY = `
+{"messageType": "AdvisorySituationData",
+  "encodeType": "hex",
+  "encodedMsg": "C440000000000000000711413AB8271E820616385BEB89C470CC45942BD4A00000000203282003F027AC0032C4FDE7F8EC962A1F8330C0056138DAB978B1E251AFFFFC46589FE805080FC271B572F163C4A3580325FFFE215E2A28885001CD78D638D0BD1F27E189112834053703D02F9A3EE0EA8178A0CDAA4B62B86590256C9C362EDA992A1BAD81766B0D3A76B52286189D50C0C321F030D1A1A7E3902B711DEE6E7E3680688283CCAC3594C998A41B4584AB820E4CA4492306F7812575022E9865C01C910082650DAD303FFC87086122B043648505DB81D480C858512218A4769D08741CD11CA048857A1D34601ED37078C912244FA7B9B789228213D587848AA641E6A241E52C4E1AB12335907FB23091314B3F3E9048DCA51E64A22480854F7487122FD467F689891AB623D5170492662A03D0E1486163BE0FDCAE851B87B937460CC3DF321D21A1454152980FEC94805C85C5BD42182C628BC4600190744C6258221000"}`;
+
+const DEFAULT_GETALLITISCODES_QUERYARGUMENTS = ``;
+
+const DEFAULT_WZDX_POSTALABBREVIATION_QUERYARGUMENTS =
+    `?
+postalabbreviation=MA`;
+
+const DEFAULT_WZDX_SPECIFICLOCATION_QUERYARGUMENTS =
+    `?
+lat=37.594049901
+&
+lon=-122.490431
+&
+dist=5`;
+
+export const REQUESTOBJECTS = [
+    {
+        displayText: "GetData",
+        request: "GetData",
+        requestType: "POST",
+        defaultQueryOrBody: DEFAULT_EXECUTEQUERYASYNC_BODY,
+        notes: "Alternatives to GetData are GetBundles and GetDistributions."
+    },
+    {
+        displayText: "GetDecodedMessages",
+        request: "GetDecodedMessages",
+        requestType: "POST",
+        defaultQueryOrBody: DEFAULT_EXECUTEQUERYASYNC_BODY,
+        notes: "Returns advisory messages in a human-readable format."
+    },
+    {
+        displayText: "GetGeoJsonData",
+        request: "GetGeoJsonData",
+        requestType: "POST",
+        defaultQueryOrBody: DEFAULT_EXECUTEQUERYASYNC_BODY,
+        notes: "Returns advisory messages as a GeoJSON FeatureCollection with ITIS codes."
+    },
+    {
+        displayText: "GetDataByRecordId",
+        request: "GetDataByRecordId",
+        requestType: "POST",
+        defaultQueryOrBody: DEFAULT_GETDATABYRECORDID_POST_BODY,
+        notes: "Type is 0 for hex and 1 for decimal."
+    },
+    {
+        displayText: "GetMessagesBetweenPlaces",
+        request: "GetMessagesBetweenPlaces",
+        requestType: "GET",
+        defaultQueryOrBody: DEFAULT_GETMESSAGESBETWEENPLACES_QUERYARGUMENTS,
+        notes: "The default road is Sheridan, WY to Cheyenne, WY and should have some messages."
+    },
+    {
+        displayText: "GetMessagesIntersectingGeometry",
+        request: "GetMessagesIntersectingGeometry",
+        requestType: "POST",
+        defaultQueryOrBody: DEFAULT_GETMESSAGESINTERSECTINGGEOMETRY_BODY,
+        notes: "The default covers an area from Denver northward and should have some messages."
+    },
+    {
+        displayText: "GetAllITISCodes",
```

Comment:
I'm not sure we need to expose this one - we're really just using that for an internal query from our web application to help display linestrings/colors/translations/etc.

Reply:
The reason I thought that we might want this one is because a DOT (or some other User) might not know which ITIS codes we accepted.  This came up recently when it turned out that Wyoming was using 'Left' and 'Right' but we didn't accept those two codes.  (There is a PR in progress to accept 'Left' and 'Right'.)
Removed but let me know if I have convinced you and you want it back.